scripts/sdt.coffee

```diff
@@ -1,46 +1,152 @@
-class Schedule
-  constructor: (@talks)->
-  toString: ->
-    @talks.map((talk)->
-      "- *#{talk.title}* _by #{talk.speaker.name}_"
-    ).join("\n")
+# Description:
```

Comment:
I didn't know this commenting style for CoffeeScript. I've tried it, unfortunately, this doesn't work well with [TomDoc](http://tomdoc.org/) which is used by [hubot-help](https://github.com/hubot-scripts/hubot-help) to parse help text for hubot bots.